### ISSUE TITLE
UX: No user feedback while the preview link loads

### ISSUE THREAD

**Issue by NightJar** [silverstripe/silverstripe-sharedraftcontent#76]
When using the CMS as a content editor, when I choose to share a preview URL with someone else I click the "Share draft" link/popup in the bottom right of the page editor, which will provide me with the public URL for this page's draft.

However; while that URL is being generated, which may be some time, there is no user feedback and it appears like text input field. This is confusing, and I tried to type something in there thinking some input was needed to form the URL.

It would be neat if some kind of spinner was shown and/or a message so the user knows what's going on
>Please wait while the URL is generated

**Comment by robbieaverill** [comment]
I'd probably be more in favour of a loading spinner or something, rather than a message (personally). I suspect this is a v3 thing as well, do we want to add that label too?

**Comment by sachajudd** [comment]
I would recommend to shorten the text to say "Loading..." until the URL appears. 🙂  

![image](https://user-images.githubusercontent.com/24258161/41576982-7f225d74-73de-11e8-8d37-86184a6e6ff2.png)

**Comment by robbieaverill** [comment]
We also have the option of the loading component in React: https://github.com/silverstripe/silverstripe-admin/pull/490

**Comment by sachajudd** [comment]
Right, I personally still prefer the text and keeping that particular component separate for loading screens rather than inputs or popovers.
